sites/public/lib/EligibilityContext.tsx

```diff
@@ -7,18 +8,71 @@ export enum AgeRangeType {
   "PreferNotSay" = "preferNotSay",
 }
 
+export enum AccessibilityFeatures {
+  "elevator"= "elevator",
+  "wheelchairRamp"= "wheelchairRamp",
+  "serviceAnimalsAllowed"= "serviceAnimalsAllowed",
+  "accessibleParking"= "accessibleParking",
+  "parkingOnSite"= "parkingOnSite",
+  "inUnitWasherDryer"= "inUnitWasherDryer",
+  "laundryInBuilding"= "laundryInBuilding",
+  "barrierFreeEntrance"= "barrierFreeEntrance",
+  "rollInShower"= "rollInShower",
+  "grabBars"= "grabBars",
+  "heatingInUnit"= "heatingInUnit",
+  "acInUnit"= "acInUnit",
+}
+
 class EligibilityRequirements {
   age: AgeRangeType
   householdSizeCount: number
   income: string
   disability: string
+  elevator: boolean
+  wheelchairRamp: boolean
+  serviceAnimalsAllowed: boolean
+  accessibleParking: boolean
+  parkingOnSite: boolean
+  inUnitWasherDryer: boolean
+  laundryInBuilding: boolean
+  barrierFreeEntrance: boolean
+  rollInShower: boolean
+  grabBars: boolean
+  heatingInUnit: boolean
+  acInUnit: boolean
   completedSections: number
 
-  constructor(age: AgeRangeType, householdSizeCount: number, income: string, disability: string) {
+  
+
+  constructor(age: AgeRangeType, householdSizeCount: number, income: string, disability: string, elevator: boolean,
+    wheelchairRamp: boolean,
+    serviceAnimalsAllowed: boolean,
+    accessibleParking: boolean,
+    parkingOnSite: boolean,
+    inUnitWasherDryer: boolean,
+    laundryInBuilding: boolean,
+    barrierFreeEntrance: boolean,
+    rollInShower: boolean,
+    grabBars: boolean,
+    heatingInUnit: boolean,
+    acInUnit: boolean) {
     this.age = age
     this.householdSizeCount = householdSizeCount
     this.income = income
     this.disability = disability
+    this.elevator = elevator
+      this.wheelchairRamp = wheelchairRamp
```

Comment:
Nit: spacing seems wrong here. But maybe prettier will take care of it?

Reply:
fixed